src/components/notification/utils.ts

```diff
@@ -0,0 +1,60 @@
+export function mapNotification(notification, channelName) {
+  const channelText = channelName ? `#${channelName}` : 'a channel';
+
+  if (notification.notificationType === 'chat_channel_mention') {
+    let senderName = displayName(notification.originUser?.profileSummary);
+
+    return {
+      id: notification.id,
+      createdAt: notification.createdAt,
+      body: `${senderName} mentioned you in ${channelText}`,
+      originatingName: senderName,
+      originatingImageUrl: notification.originUser?.profileSummary?.profileImage,
+      isUnread: notification.isUnread,
+    };
+  } else if (notification.notificationType === 'chat_channel_message_replied') {
+    let senderName = displayName(notification.originUser?.profileSummary);
+
+    return {
+      id: notification.id,
+      createdAt: notification.createdAt,
+      body: `${senderName} replied to you in ${channelText}`,
+      originatingName: senderName,
+      originatingImageUrl: notification.originUser?.profileSummary?.profileImage,
+      isUnread: notification.isUnread,
+    };
+  } else if (notification.notificationType === 'chat_dm_mention') {
+    let senderName = displayName(notification.originUser?.profileSummary);
+    return {
+      id: notification.id,
+      createdAt: notification.createdAt,
+      body: `${senderName} mentioned you in a conversation`,
+      originatingName: senderName,
+      originatingImageUrl: notification.originUser?.profileSummary?.profileImage,
+      isUnread: notification.isUnread,
+    };
+  } else if (notification.notificationType === 'chat_dm_message_replied') {
+    let senderName = displayName(notification.originUser?.profileSummary);
+    return {
+      id: notification.id,
+      createdAt: notification.createdAt,
+      body: `${senderName} replied to you in a conversation`,
+      originatingName: senderName,
+      originatingImageUrl: notification.originUser?.profileSummary?.profileImage,
+      isUnread: notification.isUnread,
+    };
```

Comment:
this is repeated (only `body` is different), maybe wrap in a function? 

```js
const send = (notification, body) => {
  return {
      id: notification.id,
      createdAt: notification.createdAt,
      body,
      originatingName: senderName,
      originatingImageUrl: notification.originUser?.profileSummary?.profileImage,
      isUnread: notification.isUnread,
    }
```    

Here you can use 
```js
return send(notification, `${senderName} replied to you in a conversation`);
```